libs/island-ui/core/src/lib/Button/Button.treat.ts

```diff
@@ -162,7 +162,14 @@ export const circleSizes = styleMap({
   },
 })
 
-type PrimaryColors = (
+type PrimaryDarkColors = (
```

Comment:
there should be no need to duplicate primaryColors type, it should be sufficient for dark and light use-cases if not it can likely be modified a little to suite more use-cases